scripts/local/e2e_test_manual.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
```

Comment:
thoughts on making this a command option in the original `e2e_test.sh` like `./scripts/local/e2e_test --local args...`?